### ISSUE TITLE
Content improvement

### ISSUE THREAD

**Issue by jillsthebest** [udacity/dlnd-issue-reports#63]
Content in section Intro to Neural Networks - Backpropagation, needs to be rewritten in a more rigorous manor for the sake of clarity. For example the term "Calculate error gradient for hidden layer" is used in the coding quiz at the bottom, while terms like, " hidden unit error" and "hidden to output weight step" are used in the explanations, making it difficult for a beginner to discern what is what. Please choose one phrase and stick with it for consistency and clarity. The explanations so far have all been a bit too casual and could use more "mathematical" structure

**Comment by yepesasecas** [comment]
+1

**Comment by clapollo** [comment]
Redid some of these to be more consistent, but we may have missed some. I'm closing this issue, but if you find any specific cases that are still unclear, please create a new issue with the details. Thank you!
